cvmfs/cvmfs.cc

```diff
@@ -1900,7 +1903,6 @@ bool Pin(const string &path) {
         return false;
       }
       cache_manager_->Close(fd);
-      return true;
```

Comment:
This is a scope of a for-loop. So it does change functionality, because the loop is always executed once at max.